restic/internal/snapshots.py

```diff
@@ -3,9 +3,12 @@
 from restic.internal import command_executor
 
 
-def run(restic_base_command, group_by=None):
+def run(restic_base_command, snapshot_id=None, group_by=None):
     cmd = restic_base_command + ['snapshots']
 
+    if snapshot_id:
+        cmd.extend([snapshot_id])
```

Comment:
It's a little simpler to just do `cmd.append(snapshot_id)`.

Reply:
Thanks, fixed! :)